### PR TITLE
Update Stretchly to 1.11.0

### PR DESCRIPTION
**Pull request by auanasgheps**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
I have manually made this update. Can you please check why the bot failed to find it? 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/71548)

### CODE REVIEW

**Comment by auanasgheps** [comment]
I get this prompt while installing, i'm investigating.
image

**Comment by ItzLevvie** [comment]
> I get this prompt while installing, i'm investigating.

cc @auanasgheps: It's because "Microsoft Defender SmartScreen" (which is the blue prompt) has been removed from Windows Sandbox in Windows 11 which is most likely a side effect of ContainerOSPlus that Windows Sandbox uses in newer builds of Windows.

You can workaround it by running this in Windows PowerShell to disable Microsoft Defender SmartScreen:
```powershell
if ($env:USERNAME -eq "WDAGUtilityAccount") {
    if (-not(Get-ItemProperty -Path HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer -Name SmartScreenEnabled)) {
        Set-ItemProperty -Path HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Explorer -Name SmartScreenEnabled -Value Off -Force
    }
}
```

**Comment by auanasgheps** [comment]
@ItzLevvie thank you!
If this is the case, the application installs normally without any user prompt.

File: manifests/s/Stretchly/Stretchly/1.11.0/Stretchly.Stretchly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/hovancik/stretchly/releases/download/v1.11.0/Stretchly-Setup-1.11.0.exe
+  InstallerSha256: c2592e60e28aed1c66736b370bd4a5f934503c24818ad5a1015b0a9251f7542c
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/Stretchly/Stretchly/1.11.0/Stretchly.Stretchly.installer.yaml:18]:
```suggestion
- Architecture: x64
```
This application only runs on 64-bit version of Windows.

File: manifests/s/Stretchly/Stretchly/1.11.0/Stretchly.Stretchly.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/s/Stretchly/Stretchly/1.11.0/Stretchly.Stretchly.installer.yaml:12]:
```suggestion
- Architecture: x64
```
This application only runs on 64-bit version of Windows.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — DISMISSED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**